test/rpc-integration/test-questionmark-vars/response-two-ques-external.booster-dev

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
Need to rename this file, it must have a `.json` suffix so it will be picked up by the `kore-rpc-dev` test.
Same for `response-one-ques.booster.dev

Reply:
Unfortunately, we cannot reuse the same response file for `booster-dev` and `kore-rpc-dev`, because `booster-dev` returns `"abotrted"` when no rules apply (the term index is None). While it would perhaps make sense to return `"stuck"` in this case (like `kore-rpc`/`kore-rpc-dev` does), this is out of scope of this PR. Hence, I've decided to separate the responses for now...